pkg/kubelet/util/util_darwin.go

```diff
@@ -0,0 +1,79 @@
+// +build darwin
```

Comment:
Why adding this file?

Reply:
hmm, it is used for compiling in macOS, but could reuse util_unix actually. will remove it.